DemoApp/StreamChat/Components/DemoChatChannelListItemView.swift

```diff
@@ -11,7 +11,7 @@ final class DemoChatChannelListItemView: ChatChannelListItemView {
         if content?.searchResult?.message != nil {
             return super.contentBackgroundColor
         }
-        if AppConfig.shared.demoAppConfig.isChannelPinningEnabled && content?.channel.isPinned == true {
+        if content?.channel.isPinned == true {
```

Comment:
Out of scope (but not intended 😄 ) but in the future if we have global pinning, we might need to have more than 1 property to distinguish `isPinned` from local only or global only 🤔